app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -68,6 +68,18 @@ def datastores(opts = {})
       end
     end
 
+    def storage_pods(opts = {})
+      if opts[:storage_pod]
+        dc.storage_pods.get(opts[:storage_pod]) rescue {}
+      else
+        name_sort(dc.storage_pods.all()) rescue []
+      end
+    end
+
+    def available_storage_pods(storage_pod = nil)
```

Comment:
What's an `unavailable` storage pod? A full one? I'm just wondering if we need the `available` prefix here :smile: